CMakeLists.txt

```diff
@@ -62,17 +53,17 @@ list(APPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/cmake")
 
 find_package(Boost 1.84.0 CONFIG)
 find_package(cryptopp 8.9.0 MODULE)
-find_package(fmt 10.2.1 CONFIG)
-find_package(glslang 14.2.0 CONFIG)
+find_package(fmt 11.0.1 CONFIG)
```

Comment:
I would avoid needlessly updating packages all the time. Linux distros don't always support latest versions